plugins/hue/__init__.py

```diff
@@ -43,9 +43,9 @@ def __init__(self, smarthome, hue_ip = '', hue_user = '', hue_port = '80', cycle
         # parameter zu übergabe aus der konfiguration pulgin.conf
         self._sh = smarthome
         # parmeter übernehmen, aufteilen und leerzeichen herausnehmen
-        self._hue_ip = [hue_ip]
-        self._hue_user = [hue_user]
-        self._hue_port = [hue_port]
+        self._hue_ip = hue_ip
+        self._hue_user = hue_user
+        self._hue_port = hue_port
```

Comment:
Using `hue_ip = value | value | value` in configuration will result in a `list`, but configuring only one value (e.g. `hue_ip = value`) you will get a `str`.

I think checking the type and converting it into a `list` - if not already - would be a good practice.

It would be really cool when we have something like a "plugin parameter descriptor" and the plugin developer can just configure which parameter and which types he uses and do not need to bother in converting (str to list etc.) or type casting (str to int etc.). But still we do not have this and we still need to to it the hard way in all plugins :smiley: